Sources/QueryRef.swift

```diff
@@ -17,11 +17,12 @@ import Foundation
 import Combine
 import Observation
 
+/// The type of publisher to use for the Query Ref
 @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
 public enum ResultsPublisherType {
-  case auto // automatically determine ObservableQueryRef
-  case observableObject // pre-iOS 17 ObservableObject
-  case observableMacro // iOS 17+ Observation framework
+  case auto /// automatically determine ObservableQueryRef
```

Comment:
Does this work when you build the docs in Xcode? I thought the comments needed to be on the line above?

Reply:
Thanks for helping out with the docs build. Yes, this seems to work except for the first enum case. I've changed it to where the comments are at the top.